golint/golint.go

```diff
@@ -114,14 +127,23 @@ func lintImportedPackage(pkg *build.Package, err error) {
 
 	var files []string
 	files = append(files, pkg.GoFiles...)
-	files = append(files, pkg.CgoFiles...)
```

Comment:
You can test it with a package consisting of a single file such as this:

```
package foo

import "C"

func f_o(){}
```

Behaviour without my addition: 

```
dominikh-pc /tmp/foo $ golint .
dominikh-pc /tmp/foo $ golint *.go
foo.go:5:6: don't use underscores in Go names; func f_o should be fO
```

Behaviour with my addition:

```
dominikh-pc /tmp/foo $ golint .
foo.go:5:6: don't use underscores in Go names; func f_o should be fO
dominikh-pc /tmp/foo $ golint *.go
foo.go:5:6: don't use underscores in Go names; func f_o should be fO
```